packages/docusaurus-theme-common/src/components/Details/styles.module.css

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/*
+CSS variables, meant to be overriden by final theme
+ */
+.details {
+  --details-summary-arrow-size: 0.5rem;
+  --details-transition: transform 200ms ease;
+  --details-decoration-color: grey;
+}
+
+.details > summary {
+  position: relative;
+  cursor: pointer;
+  list-style: none;
+  margin-left: 1.8rem;
+}
+
+.details > summary::-webkit-details-marker {
+  display: none;
+}
+
+.details > summary:before {
+  position: absolute;
+  content: '';
+  top: 0.25rem;
```

Comment:
I don't like the After, for me the goal is to have the center of he arrow aligned with the center of the text, like the native UI:

![image](https://user-images.githubusercontent.com/749374/127142116-8ba5b9fa-fbba-460c-be98-afc7d8ed593b.png)

Instead of applying your change, I'm going to reduce a bit the arrow size so that it is closed to the native size and feels more aligned in top/bottom/center at the same time